### PR TITLE
chore: more logs for axios and passthrough for debuggability

### PR DESCRIPTION
**Pull request by tomkit**
log axios error responses for remote providers:
```
supaglue-api-1          | [22:42:15.316] WARN: axios request failed
supaglue-api-1          |     context: {
supaglue-api-1          |       "applicationEnv": "development",
supaglue-api-1          |       "applicationName": "My Application",
supaglue-api-1          |       "applicationId": "a5398e5c-3f64-4b41-9cc1-cffeee9dfb0a",
supaglue-api-1          |       "orgId": "org_2OO0cz8xFHmux421XdlbnWQ9OrS"
supaglue-api-1          |     }
supaglue-api-1          |     err: {
supaglue-api-1          |       "message": "Request failed with status code 400",
supaglue-api-1          |       "name": "AxiosError",
supaglue-api-1          |       "stack":
supaglue-api-1          |           AxiosError: Request failed with status code 400
supaglue-api-1          |               at settle (/app/node_modules/axios/lib/core/settle.js:19:12)
supaglue-api-1          |               at IncomingMessage.handleStreamEnd (/app/node_modules/axios/lib/adapters/http.js:556:11)
supaglue-api-1          |               at IncomingMessage.emit (node:events:529:35)
supaglue-api-1          |               at IncomingMessage.emit (node:domain:489:12)
supaglue-api-1          |               at endReadableNT (node:internal/streams/readable:1368:12)
supaglue-api-1          |               at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
supaglue-api-1          |       "config": {
supaglue-api-1          |         "transitional": {
supaglue-api-1          |           "silentJSONParsing": true,
supaglue-api-1          |           "forcedJSONParsing": true,
supaglue-api-1          |           "clarifyTimeoutError": false
supaglue-api-1          |         },
supaglue-api-1          |         "adapter": [
supaglue-api-1          |           "xhr",
supaglue-api-1          |           "http"
supaglue-api-1          |         ],
supaglue-api-1          |         "transformRequest": [
supaglue-api-1          |           null
supaglue-api-1          |         ],
supaglue-api-1          |         "transformResponse": [
supaglue-api-1          |           null
supaglue-api-1          |         ],
supaglue-api-1          |         "timeout": 0,
supaglue-api-1          |         "xsrfCookieName": "XSRF-TOKEN",
supaglue-api-1          |         "xsrfHeaderName": "X-XSRF-TOKEN",
supaglue-api-1          |         "maxContentLength": -1,
supaglue-api-1          |         "maxBodyLength": -1,
supaglue-api-1          |         "env": {},
supaglue-api-1          |         "headers": {
supaglue-api-1          |           "Accept": "application/json, text/plain, */*",
supaglue-api-1          |           "Content-Type": "application/json",
supaglue-api-1          |           "Authorization": "[Redacted]",
supaglue-api-1          |           "User-Agent": "axios/1.3.4",
supaglue-api-1          |           "Content-Length": "24",
supaglue-api-1          |           "Accept-Encoding": "gzip, compress, deflate, br"
supaglue-api-1          |         },
supaglue-api-1          |         "params": {
supaglue-api-1          |           "count": 100
supaglue-api-1          |         },
supaglue-api-1          |         "method": "post",
supaglue-api-1          |         "url": "https://api.hubapi.com/contacts/v1/lists",
supaglue-api-1          |         "data": "{\"data\":{\"asdf\":\"asdf\"}}"
supaglue-api-1          |       },
supaglue-api-1          |       "code": "ERR_BAD_REQUEST",
supaglue-api-1          |       "status": 400
supaglue-api-1          |     }
```

log customer body for passthrough for non-2xx:
```
supaglue-api-1          | [23:22:52.660] WARN: non-2xx passthrough axios response
supaglue-api-1          |     context: {
supaglue-api-1          |       "applicationEnv": "development",
supaglue-api-1          |       "applicationName": "My Application",
supaglue-api-1          |       "applicationId": "a5398e5c-3f64-4b41-9cc1-cffeee9dfb0a",
supaglue-api-1          |       "orgId": "org_2OO0cz8xFHmux421XdlbnWQ9OrS",
supaglue-api-1          |       "passthrough": {
supaglue-api-1          |         "method": "POST",
supaglue-api-1          |         "path": "/crm/v3/objects/companiess?limit=1&archived=true",
supaglue-api-1          |         "headers": {
supaglue-api-1          |           "Content-Type": "application/json"
supaglue-api-1          |         }
supaglue-api-1          |       }
supaglue-api-1          |     }
supaglue-api-1          |     passthroughBody: {
supaglue-api-1          |       "headers": {
supaglue-api-1          |         "Content-Type": "application/json"
supaglue-api-1          |       },
supaglue-api-1          |       "path": "/crm/v3/objects/companiess?limit=1&archived=true",
supaglue-api-1          |       "method": "POST",
supaglue-api-1          |       "body": {
supaglue-api-1          |         "asdf": "asdf"
supaglue-api-1          |       }
supaglue-api-1          |     }
supaglue-api-1          |     passthroughResponse: {
supaglue-api-1          |       "url": "https://api.hubapi.com/crm/v3/objects/companiess?limit=1&archived=true",
supaglue-api-1          |       "status": 400,
supaglue-api-1          |       "headers": {
supaglue-api-1          |         "date": "Mon, 06 Nov 2023 23:22:52 GMT",
supaglue-api-1          |         "content-type": "application/json;charset=utf-8",
supaglue-api-1          |         "content-length": "130",
supaglue-api-1          |         "connection": "close",
supaglue-api-1          |         "cf-ray": "822114be58869453-SJC",
supaglue-api-1          |         "cf-cache-status": "DYNAMIC",
supaglue-api-1          |         "strict-transport-security": "max-age=31536000; includeSubDomains; preload",
supaglue-api-1          |         "vary": "origin, Accept-Encoding",
supaglue-api-1          |         "access-control-allow-credentials": "false",
supaglue-api-1          |         "x-content-type-options": "nosniff",
supaglue-api-1          |         "x-envoy-upstream-service-time": "16",
supaglue-api-1          |         "x-evy-trace-listener": "listener_https",
supaglue-api-1          |         "x-evy-trace-route-configuration": "listener_https/all",
supaglue-api-1          |         "x-evy-trace-route-service-name": "envoyset-translator",
supaglue-api-1          |         "x-evy-trace-served-by-pod": "iad02/hubapi-td/envoy-proxy-5b5c96c966-llhrc",
supaglue-api-1          |         "x-evy-trace-virtual-host": "all",
supaglue-api-1          |         "x-hubspot-correlation-id": "8bb02a03-0e0a-4cf7-a18c-a4927955cfe2",
supaglue-api-1          |         "x-hubspot-ratelimit-interval-milliseconds": "10000",
supaglue-api-1          |         "x-hubspot-ratelimit-max": "100",
supaglue-api-1          |         "x-hubspot-ratelimit-remaining": "99",
supaglue-api-1          |         "x-hubspot-ratelimit-secondly": "10",
supaglue-api-1          |         "x-hubspot-ratelimit-secondly-remaining": "9",
supaglue-api-1          |         "x-request-id": "8bb02a03-0e0a-4cf7-a18c-a4927955cfe2",
supaglue-api-1          |         "x-trace": "2B76F846A355167B29014691A0F879902DE755F195000000000000000000",
supaglue-api-1          |         "report-to": "{\"endpoints\":[{\"url\":\"https:\/\/a.nel.cloudflare.com\/report\/v3?s=qdg0xxdeNdBuO9Qh73eHlFpvp2BviIG1Y2JW5ta%2BuptAXnVn73%2B3oVRfmv%2BKHSnAjRa8CQoNn5%2FGJnnxGM7mjU7SKk3QJKLSxxrGKH3ZOAV3aO%2F%2BGhFVtwYSxmG9lqFL\"}],\"group\":\"cf-nel\",\"max_age\":604800}",
supaglue-api-1          |         "nel": "{\"success_fraction\":0.01,\"report_to\":\"cf-nel\",\"max_age\":604800}",
supaglue-api-1          |         "server": "cloudflare"
supaglue-api-1          |       },
supaglue-api-1          |       "body": {
supaglue-api-1          |         "status": "error",
supaglue-api-1          |         "message": "Unable to infer object type from: companiess",
supaglue-api-1          |         "correlationId": "8bb02a03-0e0a-4cf7-a18c-a4927955cfe2"
supaglue-api-1          |       }
supaglue-api-1          |     }
```
## Test Plan

[Describe test plan here]

## Deployment instructions

[Add any special deployment instructions here]

### CODE REVIEW

File: packages/core/lib/logger.ts [hunk 1]
```diff
@@ -59,7 +59,6 @@ export const logger: Logger = wrapLogger(
       base: undefined, // don't log hostname and pid,
       redact: [
         'err.config.headers.Authorization', // axios errors
-        'context.passthroughRequest.body',
```
Comment (tomkit) [inline packages/core/lib/logger.ts:62]: this wasn't doing anything

File: apps/api/services/passthrough_service.ts [hunk 2]
```diff
@@ -14,12 +13,6 @@ export class PassthroughService {
     request: SendPassthroughRequestRequest
   ): Promise<SendPassthroughRequestResponse> {
     const client = await this.#remoteService.getRemoteClient(connectionId);
-    // TODO I don't think this is actually working right now. Not seeing any context on successful request logs
-    //      and error logs are missing this key. Figure out why
-    addLogContext('passthroughRequest', {
-      path: request.path,
-      method: request.method,
-    });
```
Comment (tomkit) [inline apps/api/services/passthrough_service.ts:22]: not used/working

Comment (lucasmarshall) [reply apps/api/services/passthrough_service.ts:22]: I have seen this working and acutally used it.

Comment (asdfryan) [reply apps/api/services/passthrough_service.ts:22]: +1 -- I think this actually is working. Are you sure you weren't looking at the old passthrough lots (the one scoped under`/crm`

**Pull request merged**